frontend/src/employee-frontend/components/absences/AbsenceTable.tsx

```diff
@@ -109,32 +129,79 @@ const AbsenceTableRow = React.memo(function AbsenceTableRow({
           <DisabledCell />
         </td>
       ))}
+      {reservationEnabled && (
+        <>
+          <td>
+            <NumbersColumn>
+              {reservationTotalHours ? `${reservationTotalHours} h` : '-'}
+              <Gap size="xs" horizontal />
+              <WarningPlaceholder />
+            </NumbersColumn>
+          </td>
+          <td>
+            <NumbersColumn warning={showAttendanceWarning}>
+              {attendanceTotalHours ? `${attendanceTotalHours} h` : '-'}
+              <Gap size="xs" horizontal />
+              {showAttendanceWarning ? (
+                <FontAwesomeIcon
+                  icon={fasExclamationTriangle}
+                  size="1x"
+                  color={theme.colors.status.warning}
+                />
+              ) : (
+                <WarningPlaceholder />
+              )}
+            </NumbersColumn>
+          </td>
+        </>
+      )}
     </tr>
   )
 })
 
+const NumbersColumn = styled.div<{ warning?: boolean }>`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  flex-wrap: nowrap;
+  ${({ theme, warning }) =>
+    warning
+      ? css`
+          color: ${theme.colors.accents.a2orangeDark};
+          font-weight: ${fontWeights.semibold};
+        `
+      : ''};
```

Comment:
Short-circuit `&&` instead of ternary works, too. It's slightly less verbose and gets formatted with less indentation.